app/app.go

```diff
@@ -99,6 +100,10 @@ import (
 
 	"github.com/sei-protocol/sei-chain/utils/tracing"
 
+	aclmodule "github.com/sei-protocol/sei-chain/x/accesscontrol"
```

Comment:
we should set the order of genesis / begin / end block too https://github.com/sei-protocol/sei-chain/blob/master/app/app.go#L571

Reply:
Yessir, should be set down here